torchgeo/datasets/agb_live_woody_density.py

```diff
@@ -5,15 +5,14 @@
 
 import json
 import os
-from collections.abc import Iterable
```

Comment:
`typing.Iterable` is deprecated, should use `collections.abc.Iterable` instead: https://docs.python.org/3/library/typing.html#typing.Iterable